blueprints/examples/src/tests.rs

```diff
@@ -0,0 +1,103 @@
+use crate::examples::eigen_context;
+use crate::examples::eigen_context::ContextExampleTaskManager;
+use alloy_provider::Provider;
+use blueprint_test_utils::eigenlayer_test_env::{
+    AVS_DIRECTORY_ADDR, DELEGATION_MANAGER_ADDR, OPERATOR_STATE_RETRIEVER_ADDR,
+    REGISTRY_COORDINATOR_ADDR, STRATEGY_MANAGER_ADDR,
+};
+use blueprint_test_utils::helpers::get_receipt;
+use blueprint_test_utils::incredible_squaring_helpers::start_default_anvil_testnet;
+use blueprint_test_utils::{inject_test_keys, KeyGenType};
+use gadget_sdk::config::{ContextConfig, GadgetCLICoreSettings, Protocol};
+use gadget_sdk::info;
+use gadget_sdk::logging::setup_log;
+use gadget_sdk::runners::eigenlayer::EigenlayerConfig;
+use gadget_sdk::runners::BlueprintRunner;
+use gadget_sdk::utils::evm::get_provider_http;
+use reqwest::Url;
+use std::net::IpAddr;
+use std::path::Path;
+use std::str::FromStr;
+
+#[tokio::test]
+async fn test_demonstrate_eigenlayer_context() {
```

Comment:
The naming is very poor. The test eigenlayer context can test a `handle_task` function. It's ok! Make it clear and human understandable. We're putting these things in the docs and communicating with humans.
```suggestion
async fn test_eigenlayer_context() {
```